lib/web_ui/dev/firefox_installer.dart

```diff
@@ -36,6 +38,17 @@ Future<BrowserInstallation> getOrInstallFirefox(
 
   infoLog ??= io.stdout;
 
+  // When running on LUCI, if we specify the "firefox" dependency, then the
+  // bot will download Firefox from CIPD and place it in a cache and set the
+  // environment variable FIREFOX_EXECUTABLE.
+  if (io.Platform.environment.containsKey(_firefoxExecutableVar)) {
+    infoLog.writeln('Using Firefox from cache: ${io.Platform.environment[_firefoxExecutableVar]}');
```

Comment:
nit: I'd replace the word "cache" with `$_firefoxExecutableVar variable` for the same reason as Chrome above.

Reply:
Done.